website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -111,40 +111,81 @@ models:
 
 ## Materialized views
 
-The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html).
-Redshift-specific configuration includes the typical `dist`, `sort_type`, `sort`, and `backup`.
-For materialized views, there is also the `auto_refresh` setting, which allows Redshift to [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you.
-The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
-There are also some limitations that we hope to address in the next version.
+The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html)
+with the following configuration parameters:
 
-### Monitored configuration changes
+| Parameter                 | Type         | Required | Default                | Change Monitoring Support | Reference                                       |
+|---------------------------|--------------|----------|------------------------|---------------------------|-------------------------------------------------|
+| `on_configuration_change` | STRING       | NO       | `'apply'`              | N/A                       |                                                 |
+| `dist`                    | STRING       | NO       | `'EVEN'`               | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+| `sort`                    | LIST[STRING] | NO       |                        | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+| `sort_type`               | STRING       | NO       | `'AUTO'` if no `sort`  | DROP/CREATE               | [Sortkey / Distkey](#using-sortkey-and-distkey) |
+|                           |              |          | `'COMPOUND'` if `sort` |                           |                                                 |
+| `auto_refresh`            | BOOLEAN      | NO       | `False`                | ALTER                     | [Auto refresh](#auto-refresh)                   |
+| `backup`                  | BOOLEAN      | HO       | `True`                 | N/A                       | [Backup](#backup)                               |
```

Comment:
The table can be one column thinner if doing a hyperlink like [`dist`](#using-sortkey-and-distkey) in the first column instead.

Reply:
done